### ISSUE TITLE
Bold doesn't work in basic configuration

### ISSUE THREAD

**Issue by oduvan** [nitely/Spirit#60]
Our users bring the discussion here https://forum.empireofcode.com/topic/267/striped-words/

How can we fix it so user can see a difference between bold and normal font?

**Comment by oduvan** [comment]
I would also underline link since there is not a big color difference between them.

**Comment by nitely** [comment]
I fail to reproduce this. I see bold text just fine in Firefox and Chrome. What browser has this issue?

**Comment by oduvan** [comment]
Chrome on OsX for me and for this guy :) http://forum.empireofcode.com/comment/1225/find/

**Comment by nitely** [comment]
Can you try replacing the font-face in the `/static/spirit/stylesheets/styles.css` file?

Replace this:

```css
@font-face {
  font-family: ubuntu-light-webfont;
  font-weight: normal;
  font-style: normal;
  src: url("vendors/fonts/ubuntu-light-webfont.eot");
  src: url("vendors/fonts/ubuntu-light-webfont.eot?#iefix") format("embedded-opentype"), url("vendors/fonts/ubuntu-light-webfont.woff") format("woff"), url("vendors/fonts/ubuntu-light-webfont.ttf") format("truetype"), url("vendors/fonts/ubuntu-light-webfont.svg#ubuntu-light-webfont") format("svg"); }
```

by:

```css
@font-face {
  font-family: 'ubuntu-light-webfont';
  font-weight: normal;
  font-style: normal;
  src: url("vendors/fonts/ubuntu-light-webfont.woff") format("woff");
}

```

**Comment by nitely** [comment]
I applied the above fix to Spirit 0.4.3, not sure if it fixes the issue since I'm not a MacOS user. Let me know if the problem persist.

I'll replace the current Ubuntu font by the one at https://github.com/alfredxing/brick/tree/gh-pages/_fonts/ubuntu (300.woff), that may fix it.

I would appreciate if someone with a Mac can test it at http://community.spirit-project.com/topic/266/testing-the-font/

Closing this. Feel free to re-open it
